explorer/utils.py

```diff
@@ -238,9 +238,30 @@ def get_s3_connection():
                              app_settings.S3_SECRET_KEY,
                              default_bucket=app_settings.S3_BUCKET)
 
+
 def compare_sql(old_sql, new_sql):
     """
     Compares whether two sql queries are the 
     same after formatting them
     """
     return fmt_sql(old_sql) == fmt_sql(new_sql)
+
+
+def check_replication_lag():
+    """
+    Check if a replication lag exists
+    :returns: True and the replication lag interval if it 
+              exceeds 3 minutes, else returns False and None
+    """
+    conn = get_connection()
+    cursor = conn.cursor()
+
+    cursor.execute("SELECT now() - pg_last_xact_replay_timestamp() AS replication_lag")
+    replication_lag = cursor.fetchone()[0]
+
+    threshold_value = datetime.timedelta(minutes=3)
```

Comment:
3 can be a constant somewhere

Reply:
I've changed it to a global constant at the top of the file. Creating a separate constants file seems unnecessary for just one variable.